docs/tutorials/dcdi/sniffing/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+
+A customer walks into a store. Do the steps to interact with them:
+
+- Get a correct greeter
+
+- Interact with them
+
+Simple wired application:
+
+- Settings that say what punctuation to use
+
+- Registry
+
+- A bundled Greeter and Customer
+
+- An add-on which defines a FrenchGreeter and FrenchCustomer
+
+- A Datastore which stores/retrieves instances of Customers
+
+- Request and View factories to assemble the full processing chain
+
+"""
+from typing import List
+
+from wired import ServiceRegistry, ServiceContainer
+from .models import (
+    Customer, Datastore, Greeter, Resource, Request, Settings, Url, View
+)
+from .utils import (
+    process_request,
+    register_dataclass)
+
+
+def app_bootstrap(settings: Settings) -> ServiceRegistry:
+    # Make the registry
+    registry = ServiceRegistry()
+
+    # Store the settings in the registry so things later can
+    # get to them.
+    registry.register_singleton(settings, Settings)
+
+    # Make a container to use during the initialization phase
+    container: ServiceContainer = registry.create_container()
```

Comment:
unused

Reply:
I'll push a removal. Thanks for reviewing carefully.